python/ray/experimental/gpu_object_manager/gpu_object_store.py

```diff
@@ -129,12 +131,13 @@ class GPUObjectStore:
     """
 
     def __init__(self):
-        # A dictionary that maps from an object ID to a list of tensors.
+        # A dictionary that maps from an object ID to a queue of tensor lists.
+        # Each entry in the queue is a list of tensors for one copy of the object.
         #
-        # Note: Currently, `gpu_object_store` is only supported for Ray Actors.
```

Comment:
>         # A dictionary that maps from an object ID to a queue of tensor lists.
>         # Each entry in the queue is a list of tensors for one copy of the object.
This comment is for `self._gpu_object_store`, so please move L139-L140 back to the old position.